content/en/blog/_posts/2023-12-05-image-filesystem.md

```diff
@@ -0,0 +1,171 @@
+---
+layout: blog
+title: 'Image Filesystem: Configuring Kubernetes to store containers on a separate filesystem'
+date: 2023-12-05
+slug: kubernetes-separate-imagefilesystem
+---
+
+**Authors:** Kevin Hannon (Red Hat)
+
+A common issue in running/operating Kubernetes clusters is running out of disk space.
+When the node is provisioned, you should aim to have a good amount of storage space for the container runtime.  
+The container runtime usually writes to the `/var` partition.
```

Comment:
The `/var` is a path that might be a separate mount point - often is not, and where running out of disk space can have more disastrous consequences.

Reply:
Good call out. Will rephrase.

Reply:
Updated. PTAL.